pkg/kubectl/cmd/cmd.go

```diff
@@ -235,6 +236,7 @@ Find more information at https://github.com/kubernetes/kubernetes.`,
 	cmds.AddCommand(NewCmdVersion(f, out))
 	cmds.AddCommand(NewCmdExplain(f, out))
 	cmds.AddCommand(NewCmdConvert(f, out))
+	cmds.SetOutput(os.Stdout)
```

Comment:
out is already stdout: https://github.com/kubernetes/kubernetes/blob/8e81025030c47578527ad4825594e35bbb567bcb/cmd/kubectl/app/kubectl.go#L31